allennlp/predictors/text_classifier.py

```diff
@@ -0,0 +1,30 @@
+from overrides import overrides
+
+from allennlp.common.util import JsonDict
+from allennlp.data import DatasetReader, Instance
+from allennlp.models import Model
+from allennlp.predictors.predictor import Predictor
+
+
+@Predictor.register('text_classifier')
+class TextClassifierPredictor(Predictor):
+    """
+    Predictor for any model that takes in a sentence and returns
+    a single class for it.  In particular, it can be used with
+    the :class:`~allennlp.models.basic_classifier.BasicClassifier` model
+    """
+    def __init__(self, model: Model, dataset_reader: DatasetReader) -> None:
+        super().__init__(model, dataset_reader)
+
+    def predict(self, sentence: str) -> JsonDict:
+        return self.predict_json({"sentence": sentence})
+
+    @overrides
+    def _json_to_instance(self, json_dict: JsonDict) -> Instance:
+        """
+        Expects JSON that looks like ``{"sentence": "..."}``.
+        Runs the underlying model, and adds the ``"words"`` to the output.
```

Comment:
Could you correct this comment - it doesn't look like you are adding `words` to the output.

Reply:
Fixed it, will collate all changes requested and push new commit.